samcli/lib/providers/api_collector.py

```diff
@@ -184,6 +189,16 @@ def get_api(self) -> Api:
         api.stage_name = self.stage_name
         api.stage_variables = self.stage_variables
         api.cors = self.cors
+
+        for authorizers in self._authorizers_per_resources.values():
+            if len(authorizers):
+                message = f"""{os.linesep}AWS SAM CLI cannot guarantee one-to-one behaviour of Lambda authorizers 
```

Comment:
Rewording a bit:
```
AWS SAM CLI cannot guarantee 100% Fidelity between authorizers locally vs authorizers deployed on AWS. Any application critical behavior should be validated before deploying to production.
```

Reply:
Should we include a snippet about testing in the cloud with `sam sync` as an option for "True" validation? Thinking it could be a way to bring some attention to what `sam sync` can do.

Reply:
Thats a good call out, we can amend the prompt to remove personal pronouns if the Daniel's suggestion sounds good to everyone else.